lib/utils.js

```diff
@@ -291,6 +291,9 @@ exports.autoStoreData = function (req, res, next) {
 
 exports.handleCookies = function (app) {
   return function handleCookies (req, res, next) {
+    // do not show cookie banner on cookie consent page
+    app.locals.shouldShowNewCookieMessage = req.path !== '/cookie-consent'
+
     const COOKIE_NAME = 'seen_cookie_message'
```

Comment:
As the purpose of this prototype is to demonstrate the look and feel of the banner I'd suggest we change this condition.

I'd set up a page identical to index.html except that it doesn't show the cookie banner.